sklearn/metrics/scorer.py

```diff
@@ -290,7 +307,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     Examples
     --------
     >>> from sklearn.metrics import fbeta_score, make_scorer
-    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
+    >>> ftwo_scorer = make_scorer(fbeta_score, name='fbeta_score', beta=2)
```

Comment:
That's one of a few problems with a `__name__` attribute. While it can be overridden, it's just too magic and internal to be useful, but it's also insufficient: here `__name__` would be `fbeta_score`, but we need it to be something like `fbeta_score(beta=2)`